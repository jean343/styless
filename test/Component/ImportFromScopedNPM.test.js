import React from 'react';
import renderer from 'react-test-renderer';
import styled from "styled-components";
import 'jest-styled-components';

test('Support import from scoped npm', () => {
    const Div = styled.div`
        @import "./import-scoped";
        color: @foo;
	`;
    expect(renderer.create(<Div/>).toJSON()).toMatchSnapshot();
});