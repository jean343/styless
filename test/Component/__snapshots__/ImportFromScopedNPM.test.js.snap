// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Support import from scoped npm 1`] = `
.c0 {
  color: yellow;
}

<div
  className="c0"
/>
`;
